app/assets/javascripts/categorizations.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
If there is no CSS in these files, then they are not needed. Additionally, you will likely be organizing your stylesheets not by model, but by feature or page.